README.md

```diff
@@ -19,14 +19,50 @@ go get gopkg.in/DataDog/dd-trace-go.v1/contrib/gorilla/mux
 
 If you installed more packages than you intended, you can use `go mod tidy` to remove any unused packages.
 
-Requires:
+### Documentation
+
+The API is documented on [godoc](https://godoc.org/gopkg.in/DataDog/dd-trace-go.v1/ddtrace) as well as Datadog's official documentation for [Tracing Go Applications](https://docs.datadoghq.com/tracing/setup/go/) and the [Continuous Go Profiler](https://docs.datadoghq.com/tracing/profiler/enabling/go). If you are migrating from an older version of the tracer (e.g. 0.6.x) you may also find the [migration document](https://github.com/DataDog/dd-trace-go/blob/v1/MIGRATING.md) we've put together helpful.
```

Comment:
https://godoc.org/gopkg.in/DataDog/dd-trace-go.v1/ddtrace is just going to redirect to pkg.go.dev, so might ask well link there (https://pkg.go.dev/gopkg.in/DataDog/dd-trace-go.v1/ddtrace).

This may also read better as a bulleted list. e.g.
```
  - [API](https://pkg.go.dev/gopkg.in/DataDog/dd-trace-go.v1/ddtrace)
  - [Tracing Go Applications](https://docs.datadoghq.com/tracing/setup/go/)
  - [Continuous Go Profiler](https://docs.datadoghq.com/tracing/profiler/enabling/go).
  - If you are migrating from an older version of the tracer (e.g. 0.6.x) you may also find the [migration document](https://github.com/DataDog/dd-trace-go/blob/v1/MIGRATING.md) we've put together helpful.
```